add_to_app/fullscreen/android_fullscreen/app/build.gradle

```diff
@@ -5,11 +5,11 @@ apply plugin: 'kotlin-android'
 apply plugin: 'kotlin-android-extensions'
 
 android {
-    compileSdkVersion 31
+    compileSdkVersion 33
     defaultConfig {
         applicationId "dev.flutter.example.androidfullscreen"
         minSdkVersion 19
-        targetSdkVersion 28
+        targetSdkVersion 33
```

Comment:
All Android projects are migrated to compile SDK and target SDK 33, which is a requirement for the Android Gradle Plugin (AGP) version 7.